test/blackbox-tests/test-cases/enabled_if/eif-arch_sixtyfour.t/run.t

```diff
@@ -0,0 +1,37 @@
+Testing %{arch_sixtyfour} in enabled_if
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.9)
+  > EOF
+
+This will always build
+  $ cat > dune << EOF
+  > (executable
+  >  (name hello)
+  >  (enabled_if (= %{arch_sixtyfour} %{arch_sixtyfour})))
+  > EOF
+
+  $ dune exec -- ./hello.exe
+  Hello, World!
+
+Testing the version guard
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.8)
+  > EOF
+
+The parser does not understand blang expressions with unknown variables and can
+only guard on single variables.
```

Comment:
I don't understand that part of the test, what does it test?

Reply:
Ok. That looks like an issue with the error message, then. What I would suggest is to just use 3.8 with the same test `(= x x)`, check that it does not build (that's the important part), and you can make a note that the error message is not great.

Reply:
Alright, I've reverted to the bad error message and left a note about what it should be instead.